js/mixins/componentWidthMixin.js

```diff
@@ -10,6 +11,7 @@ var ComponentWidthMixin = {
 
   componentDidMount: function() {
     window.addEventListener('resize', debounce(this.onResize, 150));
+    window.addEventListener(screenfull.raw.fullscreenchange, debounce(this.onResize, 150));
```

Comment:
The player won't load on iPhone with the latest html5-skin. I believe that fullscreenchange not being valid is the issue. Maybe a simple check for null is all that is needed. Could you take a look?